File: java/Main2.java
import java.io.FileWriter;
import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Paths;
import java.nio.file.StandardOpenOption;

public class Main2
{
	public static void main(String[] args) {
		long start = System.currentTimeMillis();

		for (int i = 0; i < 100000000; i++) {
			Base base = new Base();
		}

		long finish = System.currentTimeMillis();

        long elapsed = finish - start;

        try (FileWriter writer = new FileWriter("output.txt", true)) {
            writer.write(elapsed + "\n");
        } catch (IOException e) {
            e.printStackTrace();
        }
	}
}

File: kotlin/main2.kt
import java.nio.file.Files
import java.nio.file.Paths
import java.nio.file.StandardOpenOption
import kotlin.system.measureTimeMillis

class Base() {}

fun main() {
    val elapsed = measureTimeMillis {
        for (i in 0..100000000) {
            Base()
        }
    }
    val result = "$elapsed \n"
    val fout = Paths.get("output.txt")

    Files.write(
        fout,
        result.toByteArray(),
        StandardOpenOption.CREATE,
        StandardOpenOption.APPEND
    )
}
